app/bundles/LeadBundle/Form/Type/LeadType.php

```diff
@@ -209,5 +211,21 @@ public function configureOptions(OptionsResolver $resolver)
         );
 
         $resolver->setRequired(['fields', 'isShortForm']);
+
+        $resolver->addNormalizer('constraints', static function (Options $options, array $constraints): array {
+            foreach ($options['fields'] as $field) {
+                if ('email' !== $field['alias']) {
+                    continue;
+                }
```

Comment:
It would be nice if this would validate also other unique identifier fields than just email.

Reply:
See last commit.